crates/pessimistic-proof-program/Cargo.toml

```diff
@@ -9,4 +9,4 @@ pessimistic-proof = { path = "../pessimistic-proof" }
 sp1-zkvm = { git = "https://github.com/succinctlabs/sp1", tag = "v1.0.5-testnet" }
 
 [build-dependencies]
-sp1-cli = { git = "https://github.com/succinctlabs/sp1.git", tag = "v1.0.5-testnet" }
+sp1-cli = { git = "https://github.com/succinctlabs/sp1.git", tag = "v1.0.5-testnet" }
```

Comment:
It needs a break line at the end.

Also the version of sp1 shouldn't be updated?

Reply:
Done 👍